app/controllers/admin/shelters_controller.rb

```diff
@@ -2,7 +2,9 @@
   class Admin::SheltersController < ApplicationController
     def index
       @shelters = Shelter.reverse_alphabetical
+      @pending_applications = Shelter.pending_applications
```

Comment:
Good reminder that Index can serve multiple uses for the view, just as long as we're on the index!